pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -292,13 +299,19 @@ def _maybe_release_messages(self):
             except queue.Empty:
```

Comment:
Update the "Also need to check that different ack-ids were sent for each call" from the docstring for this method.

Reply:
Sorry, I don't follow? Was that meant to be posted under the other PR? (testing splitting the ACK IDs)